examples/tfengine/simple.yaml

```diff
@@ -57,7 +57,9 @@ templates:
     MONITOR:
       PROJECT_ID: "example-monitor"
       DOMAIN: "example.com"
-    ORG_POLICIES: {} # Use default org policies.
+    ORG_POLICIES:
+      PARENT_TYPE: organization
```

Comment:
Note for later: It would be good if we can support "allowed values" for some fields, such as here. It should only be one of a few possible values, but will still generate the template if it's set to i.e. "martin".

Reply:
Yea, I'm actually going to do that in https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/pull/121 once this is merged first.